src/frontend/next/src/pages/_app.tsx

```diff
@@ -1,5 +1,7 @@
 import { AppProps } from 'next/app';
 import '../styles/globals.css';
+import '../styles/empty.css';
+import '../styles/Post.css';
```

Comment:
These won't work here, needs to be globally imported.

Reply:
Whoops, those are mistakes leftover from testing.